doc/en/how-to/logging.rst

```diff
@@ -198,6 +198,9 @@ option names are:
 If you need to record the whole test suite logging calls to a file, you can pass
 ``--log-file=/path/to/log/file``. This log file is opened in write mode which
 means that it will be overwritten at each run tests session.
+Note that relative paths for the log-file location are always resolved relative
+to the current working directory, whether passed on the CLI or declared in a
+config file.
```

Comment:
```suggestion
Note that relative paths for the log-file location, whether passed on the CLI or declared in a
config file, are always resolved relative to the current working directory.
```

Reads better to me (but I'm not a native speaker so take the suggestion with a grain of salt).